app/assets/stylesheets/_tagger.scss

```diff
@@ -7,28 +8,27 @@
 }
 
 .background-highlighter {
-  position: absolute;
-  top: 0;
   color: transparent;
-  font-size: 16px;
+  font-size: 1em;
+  left: 0px;
+  margin-bottom: 0.75em;
+  padding: 0.5em 0.5em;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `0.5em` instead of `0.5em 0.5em`